admin-site/src/Button.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+export let buttonText = "";
+export let buttonType = 'btn-outline-primary btn-block';
```

Comment:
Would this be better named as `buttonStyle`?

Reply:
Yeah, sure. And any other feedback on naming conventions is more than welcome.